### ISSUE TITLE
detail about prefix tuning

### ISSUE THREAD

**Issue by yuto3o** [thunlp/OpenPrompt#70]
In prefix_tuning_template.py file
https://github.com/thunlp/OpenPrompt/blob/675545ce1f946aa186efda8e8640dbc29fd1159f/openprompt/prompts/prefix_tuning_template.py#L207

The above code pad the attention_mask for extra prompt tokens.

Why the function 'torch.zeros' is used here? Should we use 'torch.ones' here?

**Comment by ShengdingHu** [comment]
I remember the attention mask in T5 was converted into (0,0,..., -10000,...) before this block,  where 0 indicated the valid positions and -10000 indicated the pad positions. The masks will be **added** to the attention score (see [t5 source code](https://github.com/huggingface/transformers/blob/master/src/transformers/models/t5/modeling_t5.py#L504)). You can check the value of attention mask for sure.

**Comment by yuto3o** [comment]
Thank you. I have found the related code.

https://github.com/huggingface/transformers/blob/73ec4340ec651ca1fe4f8ead9206297a4d4ed79c/src/transformers/models/t5/modeling_t5.py#L930-L941

**Comment by zig-kwin-hu** [comment]
Does this mean this prefix_tuning_template can only be used for T5, not for BART?
